CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+2.0.3 / 2019-05-27
```

Comment:
Your changes will not be in the upcoming release so please create a new section for a next release.

Reply:
Done, thanks.